internal/nearcache/manager.go

```diff
@@ -47,9 +74,23 @@ func (m *Manager) GetOrCreateNearCache(name string, cfg nearcache.Config) *NearC
 	m.nearCachesMu.Lock()
 	nc, ok = m.nearCaches[name]
 	if !ok {
-		nc = NewNearCache(&cfg, m.ss)
+		nc = NewNearCache(&cfg, m.ss, m.lg, m.doneCh)
 		m.nearCaches[name] = nc
 	}
 	m.nearCachesMu.Unlock()
 	return nc
 }
+
+func (m *Manager) RepairingTask() *ReparingTask {
+	return m.rt
+}
+
+func (m *Manager) GetNearCacheStats() []proto.Pair {
```

Comment:
Why not return the exact type but []proto.Pair here?
Something like this:

`func (m *Manager) GetNearCacheStats()  []NearCacheStat`
```
struct NearCacheStat {
    name string
    stats nearcache.Stats
}
```

Reply:
`stats.Service` should not import `inearcache`, since that creates an import cycle. I would have to put `NearCacheStat` somewhere else, unrelated. I think current approach is OK for now. It is used in a single place and not in the hot path.